client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -178,9 +200,14 @@ class ProductPurchaseFeaturesList extends Component {
 	}
 
 	getJetpackBusinessFeatures() {
-		const { selectedSite } = this.props;
-
+		const { isAutomatedTransfer, isPlaceholder, selectedSite } = this.props;
 		return [
+			<HappinessSupportCard
+				isJetpack={ !! selectedSite.jetpack && ! isAutomatedTransfer }
+				isPlaceholder={ isPlaceholder }
+				showLiveChatButton={ true }
+				liveChatButtonEventName={ 'my_plan_jetpack_professsional' }
+			/>,
 			<BusinessOnboarding
```

Comment:
I'm seeing some unexpected spacing for a .com site on a Premium plan:

![](https://cldup.com/emDLmpS9HT.png)

Reply:
I am also seeing this on my Premium dotcom test site. But I don't see it for Professional or Personal.

Reply:
@joanrho could you please try on master and see whether the line is appearing there? Thanks!

Reply:
I have just checked that there is no line atm on current master, so it is likely something introduced here. I'll check what might be causing it 👍 thanks

Reply:
I have tested on a .com Premium site and do not see the line.... 🤔

Reply:
Just tried on other plans too - seems to happen only on .com Premium...